app_legacy/Community/Controllers/UserGameListController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LegacyApp\Community\Controllers;
+
+use Illuminate\Support\Carbon;
+use LegacyApp\Http\Controller;
+use LegacyApp\Site\Models\User;
+
+class UserGameListController extends Controller
+{
+    public static function getUserSetRequestsInformation(User $user): array
```

Comment:
Not sure if this should be on a Controller. Do you have any better suggestions?

Reply:
It should not be on a controller, those ideally only should contain slim action methods. Also, we should avoid static methods whenever possible (we might want to switch to Octane at some point which does not play well with static methods and properties).

A common pattern would be to introduce `Service` classes or `Repositories` for such things, I guess moving it just an `Action` class or a global helper method would suffice for now?

Reply:
Oh, or put it on a Model directly.´

Reply:
This would make sense on a UserGameList model, but there is no list-level model. It's just a collection of UserGameListEntry models. I'll move it there for now.

Reply:
Sgtm!